src/components/Modals/Send/Form.tsx

```diff
@@ -86,16 +102,83 @@ export const Form: React.FC<SendFormProps> = ({ initialAssetId, input = '', acco
     name: SendFormFields.AssetId,
     control: methods.control,
   })
+  const formAccountId = useWatch<SendInput, SendFormFields.AccountId>({
+    name: SendFormFields.AccountId,
+    control: methods.control,
+  })
+  const formAmountCryptoPrecision = useWatch<SendInput, SendFormFields.AmountCryptoPrecision>({
+    name: SendFormFields.AmountCryptoPrecision,
+    control: methods.control,
+  })
+
+  const handleClose = useCallback(() => {
+    send.close()
+    qrCode.close()
+  }, [qrCode, send])
 
   const handleSubmit = useCallback(
     async (data: SendInput) => {
       const txHash = await handleFormSend(data, false)
       if (!txHash) return
       mixpanel?.track(MixPanelEvent.SendBroadcast)
       methods.setValue(SendFormFields.TxHash, txHash)
-      navigate(SendRoutes.Status)
+
+      dispatch(
+        actionSlice.actions.upsertAction({
+          id: txHash,
+          type: ActionType.GenericTransaction,
+          transactionMetadata: {
+            displayType: GenericTransactionDisplayType.SEND,
+            txHash,
+            chainId: fromAccountId(formAccountId).chainId,
+            accountId: formAccountId,
+            assetId,
+            amountCryptoPrecision: formAmountCryptoPrecision,
+            message: 'modals.send.status.pendingBody',
+          },
+          status: ActionStatus.Pending,
+          createdAt: Date.now(),
+          updatedAt: Date.now(),
+        }),
+      )
+
+      toast({
+        id: txHash,
+        duration: isDrawerOpen ? 5000 : null,
+        status: 'success',
+        render: ({ onClose, ...props }) => {
+          const handleClick = () => {
+            onClose()
+            openActionCenter()
+          }
+
+          return (
+            <SendNotification
+              // eslint-disable-next-line react-memo/require-usememo
+              handleClick={handleClick}
+              actionId={txHash}
+              onClose={onClose}
+              {...props}
+            />
+          )
+        },
+      })
```

Comment:
The handleClick thing is the reason why I would love to have a master hook for action center notifications that would obfuscate this thing, because we are going to duplicate these handleClick everywhere, but hey, not a big deal